components/html-block/html-block.js

```diff
@@ -118,10 +118,15 @@ export const htmlBlockContentStyles = css`
 
 let renderers;
 
-const getRenderers = () => {
+const getRenderers = async() => {
 	if (renderers) return renderers;
-	const tempRenderers = requestInstance(document, 'html-block-renderers');
-	const defaultRenderers = [ new HtmlBlockMathRenderer(), new HtmlBlockCodeRenderer() ];
+	let tempRenderers = requestInstance(document, 'html-block-renderers');
```

Comment:
You know, I think I'm being overly zealous here. Because we never hooked up the @mentions renderer, nothing is actually providing any of these here. I think I might just remove the backwards-compatible stuff.